### ISSUE TITLE
[WCAG] Language Missing/Invalid

### ISSUE THREAD

**Issue by arielkidwell** [vigetlabs/pointless-corp-astro#16]
**Details**
![Image](https://github.com/user-attachments/assets/2ee6e9eb-7a8a-4e45-b5be-7406845be380)

**Comment by ten1seven** [comment]
@arielkidwell this is fixed!
